packages/@aws-cdk/aws-ecs-patterns/lib/fargate/load-balanced-fargate-service.ts

```diff
@@ -1,11 +1,9 @@
 import ecs = require('@aws-cdk/aws-ecs');
-import { AddressRecordTarget, ARecord, IHostedZone } from '@aws-cdk/aws-route53';
-import targets = require('@aws-cdk/aws-route53-targets');
 import cdk = require('@aws-cdk/cdk');
 import { LoadBalancedServiceBase, LoadBalancedServiceBaseProps } from '../base/load-balanced-service-base';
 
 /**
- * Properties for a LoadBalancedEcsService
+ * Properties for a LoadBalancedFargateService
```

Comment:
womp womp. good catch.

Reply:
I have to give credit to @clareliguori for catching this 👍